tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -20,7 +20,7 @@
 OS_ENVIRON_ORCA_DEFAULT_BUCKET_KEY = "ORCA_DEFAULT_BUCKET"
 
 CONFIG_MULTIPART_CHUNKSIZE_MB_KEY = "s3MultipartChunksizeMb"
-CONFIG_EXCLUDE_FILE_TYPES_KEY = "excludeFileTypes"
+CONFIG_ORCA_EXCLUDED_FILE_REGEX_KEY = "OrcaExcludedFileRegEx"
```

Comment:
I like this. The typing on this always bothered be a bit. This emphasizes the complexity.